_posts/blog/2019-05-20-Ruslan-GSoC.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: post
+title: Ruslan GSoC Student
+categories: blog
+excerpt: A new member of the Whitaker lab
```

Comment:
This excerpt shows up in the blog list and so quite a lot of people see it. I think it makes sense to have GSOC in there somewhere.
```suggestion
excerpt: GSOC 2019 student in the lab working on structural covariance networks
```